bukkit/src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java

```diff
@@ -32,13 +32,13 @@
     private boolean compatSpigotBuild = false;
     private boolean spigot = true;
     private boolean lateBind = false;
-    private boolean protocolSupport = false;
```

Comment:
This still does sadly have a reason to exist

Reply:
This assignment is not necessary. JVM initializes primitives with default values, in this case, it is `false`